platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,6 +88,22 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
+### Download multiple files
+
+You can download multiple files via the user interface or using a script in your cloud provider's CLI. 
+
+:::note
+The browser download functionality currently does not work in non-Chromium browsers.
+:::
+
+Open the cloud bucket that you'd like to download and select the download icon in the **Actions** column. You can choose **Download selected** or **Download selected using code**. 
```

Comment:
See [Slack discussion](https://seqera.slack.com/archives/C06B6RRGVCZ/p1714411320345289) on removing the `Actions` column